sike-java/src/main/java/com/wultra/security/pqc/sike/math/optimized/Fp2PointProjective.java

```diff
@@ -87,12 +87,12 @@ public Fp2Point inverse() {
 
     @Override
     public Fp2Point negate() {
-        throw new RuntimeException("Not implemented yet");
+        throw new UnsupportedOperationException("Not implemented yet");
```

Comment:
Extract the string into the `Constants` class.

Reply:
Added on d4814d815887ec74358c5e52de5770447c1fdcdb.